File: DSA/Rahul_Mankani_590014725/Day31_Question2.c
#include <stdio.h>

int mul(int n) 
{
    return n << 1;
}
int main() 
{
    int n;
    printf("Enter a number: ");
    scanf("%d", &n);
    printf("%d multiplied by 2 is %d\n", n, mul(n));
    return 0;
}

File: DSA/VaniJain_590012020/Day31Question2.java
import java.util.*;
public class MultiplyByTwo {
    public static int multiplyByTwo(int n) 
  {
        return n << 1; 
  }
    public static void main(String[] args) 
  {
        Scanner sc = new Scanner(System.in);
        System.out.print("Enter a number: ");
        int n = sc.nextInt();
        int result = multiplyByTwo(n);
        System.out.println(n + " multiplied by 2 is: " + result);
  }
}
